packages/core/src/main/conversation/MessageStatusType.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+enum MessageStatusType {
+  DELIVERED = 3,
+  FAILED = 0,
+  SEEN = 4,
+  SENDING = 1,
+  SENT = 2,
+  UNSPECIFIED = -1,
+}
```

Comment:
These are webapp internal types. The protocol buffer supports these https://github.com/wireapp/generic-message-proto/blob/master/proto/messages.proto#L123-L124